src/libpgagroal/configuration.c

```diff
@@ -2395,6 +2397,29 @@ is_same_server(struct server* s1, struct server* s2)
    }
 }
 
+static bool
+is_same_tls(struct server* src, struct server* dst)
+{
+   if (src->tls != dst->tls)
+   {
+      return false;
+   }
+
+   if (!src->tls)
+   {
+      return true;
+   }
+
+   if (strcmp(src->tls_cert_file, dst->tls_cert_file) ||
+       strcmp(src->tls_key_file, dst->tls_key_file) ||
+       strcmp(src->tls_ca_file, dst->tls_ca_file))
+   {
+      return false;
+   }
+
+   return true;
```

Comment:
`return false;` should be default

Reply:
Yes I think you are right. In order to make the code style more consistent, I changed the name and logic for `is_same_tls`. The original logic is the opposite, but I forgot to change this return value😂🥺